src/main/io/gps.h

```diff
@@ -72,11 +73,19 @@ typedef enum {
     GPS_AUTOBAUD_ON
 } gpsAutoBaud_e;
 
+typedef enum {
+    UBLOX_ACK_IDLE = 0,
+    UBLOX_ACK_WAITING,
+    UBLOX_ACK_GOT_ACK,
+    UBLOX_ACK_GOT_NACK
+} ubloxAckState_e;
+
 #define GPS_BAUDRATE_MAX GPS_BAUDRATE_9600
 
 typedef struct gpsConfig_s {
     gpsProvider_e provider;
     sbasMode_e sbasMode;
+    uint8_t sbasIntegrity;
```

Comment:
Only add elements to the end of a parameter group structure. Please see:

https://github.com/betaflight/betaflight/blob/master/docs/development/ParameterGroups.md

Reply:
Sorry I keep forgetting this..